components/threading/virtualized_thread_viewer/create_comment.tsx

```diff
@@ -68,7 +70,12 @@ const CreateComment = forwardRef<HTMLDivElement, Props>(({
             <div className='channel-archived-warning__container'>
                 <BasicSeparator/>
                 <div className='channel-archived-warning__content'>
-                    <i className='icon icon-archive-outline'/>
+                    <div className='icon'>
```

Comment:
I think the wrapping `div` is not needed here anymore

Reply:
You're right, it still provides some styling that we definitely want, notably these lines from `thread_viewer.scss`:
```
.icon {
    margin-right: 3.4px;
    vertical-align: text-top;
}
```

But I could move these adjustments to the `svg` itself if that's more appropriate than a wrapping div, like this instead (so long as this is considered okay):
```
...
svg {
    margin-right: 3.4px;
    vertical-align: text-top;
}
```

Reply:
you can do that. That would be fine with me. Do we still need the `vertical-align: top`? I would rather set `display: flex;` and `align-content: center` on the parent div.

What I originally stated though was that when using `display: flex` or `display: grid` in the correct way we would not need to add those alignment/margin/padding stuff at all. For this case it is fine.

Reply:
Good idea! For both instances in this PR I removed the wrapping div from the icons, and removed the unnecessary scss class, just keeping the `margin-right` property for `svg` items in the parent scss class.

Your suggestion to use `display: flex` with `align-content: center` was a good idea, but in fact `align-content: center` was not required, however a `justify-content: center` was required in order to force the "group" back to the middle of the screen.

I think this change hits on the points you wanted addressed, so let me know if there's anything else you'd like to see adjusted or discuss about.

Thanks again for taking the time for such thorough reviews and comments.